docs/api/base-window.md

```diff
@@ -0,0 +1,1460 @@
+# BaseWindow
+
+> Create and control windows.
+
+Process: [Main](../glossary.md#main-process)
+
+This module cannot be used until the `ready` event of the `app`
+module is emitted.
+
+```javascript
+// In the main process.
+const { BaseWindow } = require('electron')
+
+const win = new BaseWindow({ width: 800, height: 600 })
+
+// TODO example
+```
```

Comment:
Should our sample snippets include `app.on('ready', () => ...)`? We mention requiring it above, but then immediately ignore that 😛

Reply:
We don't do that elsewhere in our docs, see https://www.electronjs.org/docs/latest/api/browser-window